rules/windows/credential_access_potential_lsa_memdump_via_mirrordump.toml

```
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2021/09/27"
+maturity = "production"
+updated_date = "2021/09/27"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a suspicious process access event by LSASS where the granted access value is equal to duplicate process handle
+and from an unknown call trace module. This may indicate an attempt to dump Lsass memory using MirrorDump tool in preparation
+for credential access.
```

Updates to be more similar to your description in the issue

```suggestion
Identifies suspicious access to an LSASS handle via DuplicateHandle from an unknown call trace module. 
This may indicate an attempt to bypass the NtOpenProcess API to evade detection and dump Lsass memory
for credential access.
```

May need to relint it too